### PR TITLE
Added ability to increment by a specified amount

### PR DESCRIPTION
**Pull request by laeschjs**

### CODE REVIEW

File: src/main/java/edu/rosehulman/quota/controller/IncrementQuotaController.java [hunk 1]
```diff
@@ -22,6 +25,12 @@ public Object handle(Request request, Response response) throws Exception {
     String productId = request.params(":productId");
     String userId = request.params(":userId");
     String quotaId = request.params(":quotaId");
+    String body = request.body();
```
Comment (jackhallam) [inline src/main/java/edu/rosehulman/quota/controller/IncrementQuotaController.java]: I personally think this intermediate variable is overkill

File: src/main/java/edu/rosehulman/quota/controller/IncrementQuotaController.java [hunk 2]
```diff
@@ -22,6 +25,12 @@ public Object handle(Request request, Response response) throws Exception {
...
+    String count = null;
```
Comment (jackhallam) [inline src/main/java/edu/rosehulman/quota/controller/IncrementQuotaController.java]: Save it as a BigInteger right away. Do simple error checking on it too.

File: src/main/java/edu/rosehulman/quota/controller/IncrementQuotaController.java [hunk 3]
```diff
@@ -56,8 +65,11 @@ public Object handle(Request request, Response response) throws Exception {
     }
 
     // See if adding one to the quota would go above the quota
-    BigInteger valuePlusOne = value.add(BigInteger.ONE);
-    if (valuePlusOne.compareTo(max) > 0) {
+    BigInteger newValue = value.add(BigInteger.ONE);
```
Comment (jackhallam) [inline src/main/java/edu/rosehulman/quota/controller/IncrementQuotaController.java]: Wouldn't this be double counting if `count != null`?

Comment (jackhallam) [reply src/main/java/edu/rosehulman/quota/controller/IncrementQuotaController.java]: Also maybe the variable name can be something like `incrementedValue` to be more descriptive

**Review by weinbec — APPROVED** [review]

**Pull request merged**